bindings/go/src/fdb/fdb.go

```diff
@@ -37,9 +37,9 @@ import (
 	"unsafe"
 )
 
-/* Would put this in futures.go but for the documented issue with
-/* exports and functions in preamble
-/* (https://code.google.com/p/go-wiki/wiki/cgo#Global_functions) */
+/* Would put this in futures.go but for the documented issue with */
+/* exports and functions in preamble                              */
+/* (https://code.google.com/p/go-wiki/wiki/cgo#Global_functions)  */
```

Comment:
It looks to me like other functions are consistently commented with `//` starting every line. I'm not sure why we had commented this one with a different style, but probably the best change now is to make it match the other functions.